pom.xml

```diff
@@ -264,7 +264,7 @@
                         <configuration>
                             <referenceVersion>3.25ea0</referenceVersion>
```

Comment:
We should be referencing **3.26**ea. Prob., threshold can stay on 100 then.

Reply:
Agreed - thank you - changed it